.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Open release
+
+on:
+  push:
+    branches:
+      # Matches release branches
+      - "[0-9]+.[0-9]+"
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Prepare release with Changesets
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+      - name: Install Dependencies
+        run: npm ci
+      - name: Create Release Pull Request
+        uses: changesets/action@v1
+        id: changesets
+        with:
+          title: "Prepare release ${{ github.ref_name }}"
+          commit: "Release ${{ github.ref_name }}"
+          publish: "npm run release"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Set the permissions it actually needs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions

Reply:
Disallow the rest